src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -42,27 +42,29 @@ public double getValue() {
     }
 
     /**
-     * Fetch a {@link Association} by its value.
+     * Fetch a {@link Association} by its exact value.
      *
      * @param value The friendliness value
      * @return The association related to the friendliness value. <code>null</code> if there is no {@link Association} with the provided value.
      * @author 086
      */
-    public static Association byValue(double value) {
+    public static Association byExactValue(final double value) {
         return Arrays.stream(values()).filter(association -> association.getValue() == value).findAny().orElseGet(null);
     }
 
     /**
-     * Fetch the {@link Association} closest to a friendliness value.
-     * <p>Edge cases: <code>0.5</code> becomes {@link Association#ALLY} and <code>-0.5</code> becomes {@link Association#ENEMY}</p>
+     * Fetch the {@link Association} convert value to association.
```

Comment:
This isn't an english sentence.

Reply:
yea true. didn't know how to word it better. Because I was lost thinking about how I can still leave the link there and at the same time have a fitting method name and stuff.